rllib/agents/qmix/qmix_policy.py

```diff
@@ -45,8 +48,17 @@ def __init__(self,
         self.double_q = double_q
         self.gamma = gamma
 
-    def forward(self, rewards, actions, terminated, mask, obs, next_obs,
-                action_mask, next_action_mask):
+    def forward(self,
+                rewards,
+                actions,
+                terminated,
+                mask,
+                obs,
+                next_obs,
+                action_mask,
+                next_action_mask,
+                state=None,
```

Comment:
To avoid confusing this with RNN state, could we call this `global_obs` here and elsewhere?

Reply:
how about env_state?